### PR TITLE
chore: pre-install krb5-dev and sasl2-dev libs, export JAVA_HOME in CentOS7 Dockerfile

### PR DESCRIPTION
**Pull request by GehaFearless**
issues: https://github.com/pegasus-kv/pegasus-docker/issues/13

### CODE REVIEW

File: pegasus-build-env/centos7/Dockerfile [hunk 1]
```diff
@@ -49,14 +49,20 @@ zlib-devel \
...
 flex \
+krb5-devel \
+cyrus-sasl-devel \
 patch; \
 yum clean all; \
 rm -rf /var/cache/yum;
 
 RUN pip3 install --no-cache-dir cmake
 
-ENV PATH="/opt/rh/devtoolset-7/root/bin/:${PATH}"
```
Comment (Smityz) [inline pegasus-build-env/centos7/Dockerfile:60]: why delete this line?

Comment (GehaFearless) [reply pegasus-build-env/centos7/Dockerfile:60]: look at line 66

Comment (Smityz) [reply pegasus-build-env/centos7/Dockerfile:60]: Can it be set like
```
ENV PATH="/opt/rh/devtoolset-7/root/bin/:${PATH}"
ENV PATH="/opt/maven/apache-maven-3.8.3/bin:${PATH}"
```
I think it's easier to read

File: pegasus-build-env/centos7/Dockerfile [hunk 2]
```diff
@@ -49,14 +49,20 @@ zlib-devel \
...
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz -P /opt/maven \
+    && cd /opt/maven \
+    && tar -zxf apache-maven-3.8.3-bin.tar.gz
```
Comment (Smityz) [inline pegasus-build-env/centos7/Dockerfile]:
```suggestion
    && tar -zxf apache-maven-3.8.3-bin.tar.gz \
    && rm apache-maven-3.8.3-bin.tar.gz
```

File: pegasus-build-env/centos7/Dockerfile (hunk 1 above)
Comment (GehaFearless) [reply pegasus-build-env/centos7/Dockerfile:60]: alright

**Review by acelyc111 — APPROVED** [review]

File: pegasus-build-env/centos7/Dockerfile [hunk 3]
```diff
@@ -49,14 +49,22 @@ zlib-devel \
...
 rm -rf /var/cache/yum;
 
 RUN pip3 install --no-cache-dir cmake
 
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz -P /opt/maven \
+    && cd /opt/maven \
+    && tar -zxf apache-maven-3.8.3-bin.tar.gz \
+    && rm apache-maven-3.8.3-bin.tar.gz
+
+ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.322.b06-1.el7_9.x86_64
```
Comment (acelyc111) [inline pegasus-build-env/centos7/Dockerfile:66]: Why only export JAVA_HOME in CentOS7 ?

**Pull request merged**